include/nrsc5.h

```diff
@@ -51,6 +51,16 @@
 #define NRSC5_SAMPLE_RATE_CS16_AM  46511.71875
 #define NRSC5_SAMPLE_RATE_AUDIO    44100
 
+#ifdef NRSC5_EXPORTS
```

Comment:
Is the `NRSC5_EXPORTS` conditional useful for something? It wasn't there before.

Reply:
It's to make sure dllexport is only used when building the shared library (where export is needed). dllexport isn't needed when including the header normally

Reply:
Would dllexport simply be ignored if building something other than a DLL?

Reply:
In the examples I've seen, they created a macro to determine when dllexport is used. This is most likely because `__declspec(dllimport)` should be used instead when dll isn't being built (which I can add if you'd rather). 
There is normally a huge macro to handle other platforms. Other platforms have a file instead to provide these functions.

Reply:
Am trying to be careful to not include random stuff that shouldn't be in as a public header.

Reply:
bump

Reply:
I guess that's fine, it seems like it shouldn't hurt anything.